src/example/update/constraints/NodeDetails.java

```diff
@@ -1,6 +1,6 @@
 package example.update.constraints;
 
-import example.update.initialisation.InetInitializer;
+import example.update.initialisation.CoordinatesInitializer;
```

Comment:
do a separate PR with this!

Reply:
Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one

Reply:
Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one

Reply:
Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one